server/issue_comment_handler.go

```diff
@@ -161,7 +161,7 @@ func (e *issueCommentEvent) HasE2ETest() bool {
 	return strings.Contains(strings.TrimSpace(e.Comment.GetBody()), "/e2e-test")
 }
 
-// HasE2ECancel is true if body contains "/e2e-cancel"
+// HasE2ECancel is true if body is prefixed with "/e2e-cancel"
 func (e *issueCommentEvent) HasE2ECancel() bool {
-	return strings.Contains(strings.TrimSpace(e.Comment.GetBody()), "/e2e-cancel")
+	return strings.HasPrefix(strings.TrimSpace(e.Comment.GetBody()), "/e2e-cancel")
```

Comment:
this should be a regex, maybe can be updated in the future

Reply:
Yep, sounds good.